client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -133,239 +127,236 @@ const InvestigatedPersonInfo = (props: Props) => {
-                    <Tooltip title={isSuspicionOfMutation ? mutationName ? mutationName : noInfo : ''}>
-                        <div>
-                            <PatientInfoItem testId='isSuspicionOfMutation' name='חשד למוטציה' value={isSuspicionOfMutation ? yes : noInfo} />    
+                                </>
+                                :
+                                <PatientInfoItem testId='idType' name='סוג תעודה מזהה' value={identityType}  />
+                                }
+                                {userType === UserTypeCodes.ADMIN || userType === UserTypeCodes.SUPER_ADMIN ?
+                                <>
+                                    <PatientInfoItem testId='idNumber' name='מספר תעודה מזהה' value={''} />
+                                    <Controller 
+                                        name={StaticFields.ID}
+                                        control={methods.control}
+                                        defaultValue={identityNumber}
+                                        render={(props) => (
+                                            <TextField
+                                                {...props}
+                                                disabled
+                                                className={classes.smallSizeText}
+                                                InputProps={{className: classes.smallSizeText}}
+                                                test-id={props.name}
+                                                onChange={(event) => {
+                                                    props.onChange(event.target.value as string)
+                                                    setStaticFieldsChange(true)
+                                                }}
+                                                error={get(methods.errors, props.name)}
+                                                label={get(methods.errors, props.name)?.message || ''}
+                                            />
+                                        )}
+                                    />
+                                </>
+                                :
+                                <PatientInfoItem testId='idNumber' name='מספר תעודה מזהה' value={identityNumber} />
+                                }
+                            </div>
+                            <div className={classes.line}>
+                                <PatientInfoItem testId='isDeceased' name='נפטר' value={indication((isDeceased || investigationStatus.subStatus === InvestigationComplexityByStatus.IS_DECEASED))} />
+                                {
+                                    (isDeceased ||
+                                        (investigationStatus.mainStatus === InvestigationMainStatusCodes.CANT_COMPLETE &&
+                                            investigationStatus.subStatus === InvestigationComplexityByStatus.IS_DECEASED)) &&
```

Comment:
you can move the long conditions to const with a name that will describe the codition